ui/app/src/hooks/useCurrencyFieldState.ts

```diff
@@ -6,13 +6,17 @@ const globalState = {
   fromAmount: ref<string>("0"),
   toSymbol: ref<string | null>(null),
   toAmount: ref<string>("0"),
+  priceImpact: ref<string | null>(null),
+  providerFee: ref<string | null>(null),
```

Comment:
Is this related to the field? Or do you think we should make this some kind of global state and call it something else? I am not opposed to that just think we should probably call it something else.